### ISSUE TITLE
Support for Pay to Taproot outputs and Bech32m addresses

### ISSUE THREAD

**Issue by Xekyo** [Blockchair/Blockchair.Support#567]
I'm reaching out to ask about Blockchair's plans to prepare for the upcoming Taproot softfork in November. 

As you probably know, the BIP341/BIP342 "Taproot" proposal is going to activate on Bitcoin mainnet on block 709632 (expected around November 20th, 2021). This will introduce the "Pay to Taproot" output type using the new "bech32m" address format specified in BIP350. I was wondering whether Blockchair is ready or preparing to index and display bech32m addresses by the time that Taproot activates.

**Comment by Xekyo** [comment]
To give a concrete example, in transaction [b10c007c60e14f9d087e0291d4d0c7869697c6681d979c6639dbd960792b4d41](https://blockchair.com/bitcoin/transaction/b10c007c60e14f9d087e0291d4d0c7869697c6681d979c6639dbd960792b4d41), the input addresses should be `bc1pqyqszqgpqyqszqgpqyqszqgpqyqszqgpqyqszqgpqyqszqgpqyqsyjer9e` instead of `bc1pqyqszqgpqyqszqgpqyqszqgpqyqszqgpqyqszqgpqyqszqgpqyqs3wf0qm`
and `bc1pmfr3p9j00pfxjh0zmgp99y8zftmd3s5pmedqhyptwy6lm87hf5sspknck9` instead of `bc1pmfr3p9j00pfxjh0zmgp99y8zftmd3s5pmedqhyptwy6lm87hf5ss52r5n8`.

**Comment by Har01d** [comment]
Yes, we plan to support Taproot in both the interface and the API. Not sure though if we’ll be converting existing `witness_unknown` bech32 addresses, but this shouldn’t be a big issue as there are only ~10 of them.

**Comment by Xekyo** [comment]
Great, thanks for the update.

**Comment by Xekyo** [comment]
Hi @Har01d. I just wanted to bump this issue since Taproot will be activated in a few days.

**Comment by Har01d** [comment]
Hi @Xekyo, yes, we'll be rolling out Taproot for Bitcoin, Groestlcoin, and Bitcoin Testnet in a day or two.

**Comment by Har01d** [comment]
Here's the official announcement: https://twitter.com/Blockchair/status/1458817396433731585

We'll be reindexing Bitcoin to include some of the `witness_unknown` outputs which were Taproot outputs a bit later, but this shouldn't be an issue as there's not a lot of them (see https://blockchair.com/bitcoin/outputs?q=type(witness_unknown)#)

Bitcoin Testnet is fully reindexed and is ready for testing.

**Comment by mahnunchik** [comment]
Please add `xpub` variant for Pay to Taproot for `https://api.blockchair.com/{:btc_chain}/dashboards/xpub/{:extended_key}` api endpoint.

Currently only xpub, ypub, and zpub are supported.
